File: market/integration-test/java/ru/yandex/market/logistics/lom/configuration/ClockConfiguration.java
package ru.yandex.market.logistics.lom.configuration;

import org.springframework.context.annotation.Bean;

import ru.yandex.common.util.date.TestableClock;

public class ClockConfiguration {
    @Bean
    TestableClock clock() {
        return new TestableClock();
    }
}

File: market/integration-test/kotlin/ru/yandex/market/logistics/les/configuration/ClockConfiguration.kt
package ru.yandex.market.logistics.les.configuration

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import ru.yandex.common.util.date.TestableClock

@Configuration
open class ClockConfiguration {
    @Bean
    open fun clock() = TestableClock()
}
